src/libs/actions/Report.js

```diff
@@ -471,16 +481,30 @@ function subscribeToReportCommentEvents() {
         return;
     }
 
-    Pusher.subscribe(pusherChannelName, 'reportComment', (pushJSON) => {
-        Log.info('[Report] Handled event sent by Pusher', true, {reportID: pushJSON.reportID});
-        updateReportWithNewAction(pushJSON.reportID, pushJSON.reportAction);
-    }, false,
-    () => {
+    function onResubscribeToAccountChannel() {
         NetworkConnection.triggerReconnectionCallbacks('pusher re-subscribed to private user channel');
-    })
-        .catch((error) => {
-            Log.info('[Report] Failed to initially subscribe to Pusher channel', true, {error, pusherChannelName});
-        });
+    }
+    function onChannelSubscribeFail(error) {
+        Log.info('[Report] Failed to initially subscribe to Pusher channel', true, {error, pusherChannelName});
+    }
```

Comment:
Can we remove these functions? If we're just logging a statement I don't think it's bad to repeat the log. I don't think the nested function gives us much upside here.

Reply:
Sounds good, I'll remove this :D

Reply:
Done :)